numpy/lib/tests/test_recfunctions.py

```diff
@@ -240,8 +240,8 @@ def test_structured_to_unstructured(self):
                      dtype=[('x', 'i4'), ('y', 'i4'), ('z', 'i4')])
         dd = structured_to_unstructured(d)
         ddd = unstructured_to_structured(dd, d.dtype)
-        assert_(dd.base is d)
-        assert_(ddd.base is d)
+        assert dd.view().base is d
+        assert ddd.view().base is d
 
```

Comment:
Is this necessary?

Reply:
not anymore (since the rebase), reverted.